invenio_app_rdm/records_ui/previewer/mirador.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 data-futures.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Mirador preview."""
+
+from flask import current_app, render_template
+
+def can_preview(file):
+    """Check if file can be previewed by this previewer.
+
+    :param file: The file to be previewed.
+    :returns: Boolean
+    """
+    # supported_extensions list needs . prefixed -
+    preview_extensions = current_app.config["MIRADOR_PREVIEW_EXTENSIONS"]
+    supported_extensions = ["." + ext for ext in preview_extensions]
+    return file.has_extensions(*supported_extensions)
+
+
+def preview(file):
+    """Render template.
+    """
+    ui_config = {
```

Comment:
Question:
Is there a strong reason why we don't allow this to be overridden like `preview_extensions`, enabling users to choose their own [configuration](https://github.com/ProjectMirador/mirador/blob/3c873950d03925fc13525146dd1e195fa4f073fa/src/state/actions/window.js#L63)?

Reply:
This will be moved to the config.py file, it was added here just for testing purposes as we are overriding the default config in the pre-defined mirador distribution we are currently utilizing.